### DISCUSSION TITLE
Ilia Azizi - Glassdoor+Text Mining

### DISCUSSION THREAD

**Discussion by giscus** [EllaKaye/ellakaye-quarto#6] [General]
# Ilia Azizi - Glassdoor+Text Mining

Text mining techniques applied to employer reviews from Glassdoor.com with the aim of predicting the stars given to the employers.

https://iliaazizi.com/projects/glassdoor-text/

**Comment by Unco3892** [comment]
Trying the comments here

**Reply by Unco3892** [reply]
@EllaKaye thanks for the really cool website. I used it as a template to build mine. I forgot to modify my `giscus.html` so please feel free to remove this discussion.
